### DISCUSSION TITLE
out format options with osgearth_conv

### DISCUSSION THREAD

**Discussion by christ0phd** [pelicanmapping/osgearth#2120] [Support Q&A]
Hi all,

is there any way to set file format options via oagearth_conv for the output file. Currently the tool writes out 4 samples/pixel but I want to set it to 3 to stop the TIFFReader from complaining about "Sum of Photometric type-related color channels and ExtraSamples doesn't match SamplesPerPixel. Defining non-color channels as ExtraSamples.."

Christoph

**Comment by jasonbeverage** [comment]
Hi Christoph, can you post an example of your osgearth_conv command line?  We normally use PNG, JPG or DDS as the output format from osgearth_conv so haven't ran into that since we don't export imagery as tiffs.

**Reply by christ0phd** [reply]
Hey Jason, I used the following call:
`osgearth_conv --in driver GDALImage --in url input.tif --out driver TMSImage --out url outdir/tms.xml --out format tif`
While the input.tif is 24bpp RGB image the resulting tifs in the output directory are 32bits. I tried to extend the call by `--out TIFFTAG_SAMPLESPERPIXEL 3` and by `--out TIFFTAG_PHOTOMETRIC PHOTOMETRIC_RGB`. Both additions were mentioned in the output message of osgearth_conv. But nothing changed the files in output directory.
